### PR TITLE
🇨🇳 Add Simplified Chinese locale

### PR DESCRIPTION
**Pull request by Wittoy**

### CODE REVIEW

File: src/locales/zh-CN.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "translation": {
+    "menuTitle": "隐藏播客",
+    "enabled": "启用",
+    "aggressiveMode": "增强模式",
+    "searchPageShelfAriaLabel": "想听什么？"
```
Comment (theRealPadster) [inline src/locales/zh-CN.json:6]: Just want to double check that this is the correct translation. It should be the "browse all" label on the search page here:

image

**Review by theRealPadster — CHANGES_REQUESTED** [review]
Thanks, just had the one question to make sure it works on the search page. When I test the locale on my computer I get a different translation. Just let me know if it's correct or should change and we should be good to go :)

File: src/locales/zh-CN.json (hunk 1 above)
Comment (Wittoy) [reply src/locales/zh-CN.json:6]: Aha, I understood it to be a search bar placeholder.
Preferably "浏览全部"
```suggestion
    "searchPageShelfAriaLabel": "浏览全部"
```

**Comment by theRealPadster** [comment]
Awesome thanks!

**Pull request merged**